WORKSPACE

```diff
@@ -1,57 +1,80 @@
-workspace(name = "angular_cli")
+workspace(
+    name = "angular_cli",
+    managed_directories = {"@npm": ["node_modules"]},
+)
 
-load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
 http_archive(
     name = "build_bazel_rules_nodejs",
-    sha256 = "fb87ed5965cef93188af9a7287511639403f4b0da418961ce6defb9dcf658f51",
-    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/0.27.7/rules_nodejs-0.27.7.tar.gz"],
+    # patch_args = ["-p1"],
```

Comment:
Do we want to use the patch for angular-cli?

Reply:
Removed the commented out properties.  The patch is only needed to allow zone.js to be synced into g3 which doesn't apply to the CLI repo.